pkgs/applications/science/logic/klee/default.nix

```diff
@@ -14,13 +14,42 @@
 , sqlite
 , gtest
 , lit
+
+# Clang used to build KLEE.
+, kleeClang ? clang_9
+
+# LLVM used to build KLEE.
+, kleeLLVM ? llvmPackages_9.llvm
+
+# Python used for KLEE tests.
+, kleePython ? python3.withPackages (ps: with ps; [ tabulate ])
+
+# The klee-uclibc derivation.
+, kleeuClibc ? callPackage ./klee-uclibc.nix {
+    inherit kleeClang kleeLLVM kleePython extraKleeuClibcConfig debugRuntime runtimeAsserts;
+  }
```

Comment:
Not a fan of those. It would be nicer to have them in top-level.nix or just use the default package inputs.

Reply:
I'll just move them to the let block.

Reply:
That would be cleaner, too but then you can't easily override it.

Reply:
I went ahead and used the normal inputs and moved any changes I had to make to the let block.